pkg/fusionauth/Client.go

```diff
@@ -7008,6 +7008,36 @@ func (c *FusionAuthClient) RetrieveUserByLoginIdWithContext(ctx context.Context,
 	return &resp, &errors, err
 }
 
+// RetrieveUserByLoginIdWithLoginIdTypes
```

Comment:
go doesn't support function overloading or optional params, so letting it create a separate function here.